packages/@lwc/template-compiler/src/__tests__/fixtures/attributes/attribute-uppercase/actual.html

```diff
@@ -0,0 +1,11 @@
+<template>
+    <x-button 
+        -class="r"
+        -data-xx="foo"
+        -aria-hidden="hidden"
+        -role="xx"
+        -foo-bar="x"
+        --foo-zaz="z"
```

Comment:
I feel that we should block `--`, but maybe it is just too late for this. I feel that it could be used for something else.... just a thought!

Reply:
I'm open to restrict double hyphen syntax in a different PR if we believe there are valid use cases for this; however i do want to get this work out as it is scheduled to 224 backport.